applications/inscription_indexer.md

```diff
@@ -0,0 +1,174 @@
+which can be used on any public chain within the Polkadot ecosystem. A unified inscription indexer standard benefits everyone.
+
+In the Polkadot ecosystem, Dota has the most holders among inscription projects, 
+which is undeniable. This also implies that Dota's indexer has the greatest potential to become the standard for inscription indexers in the Polkadot ecosystem, 
+and is more likely to be seen and used by more teams. It also means more developers will participate in joint construction, 
+the protocol will be continuously supplemented and improved, more people will join testing and usage, and the code will be more secure.
+
+There are other inscription projects in the Polkadot ecosystem, and they may even have their own excellent protocols, 
+but we are different. We have found that their indexers support a relatively narrow range of on-chain transactions, 
+mainly remark_with_event. In contrast, our support range is broader, covering batch, multi-signature, and delegation, 
+which makes the use cases of inscriptions more comprehensive. Under different combinations of calls in batch_all, 
+sending a memo to the chain can also interact with the chain simultaneously. In the future, system-level inscription parachains can be created (currently, the Dota team also plans to do so).
+
+The dot-20 protocol carried by the Dota indexer enables the rapid creation of more ERC20-based products.
+## Team :busts_in_silhouette:
+
+### Team members
+
+- `jyaymusk`
+- `octavei`,  `MockBL`, `lanyou`, `DOTADurial`
+
+### Contact
+
+- **Contact Name:** jyaymusk
+- **Contact Email:** jyaymusk@proton.me
+- **Website:** https://dota.fyi/
+
+### Legal Structure
+
+- **Registered Address:** Room 203, Building 10, Chuangye Road, Xixiangtang District, Nanning City, Guangxi Zhuang Autonomous Region, China.
+- **Registered Legal Entity:** N/A
+
+### Team's experience
+- `jyaymusk` Graduated from the National University of Singapore with a Master's degree in Finance. 7 years of experience in the blockchain industry.
+- `DOTADurial` 8 years of experience in the blockchain industry, with roles as an investor and product manager.
+- `octavei` 5 years experience in substrate development. Familiar with Ethereum and EOS smart contracts. Have done many blockchain projects.
+  > Thank you to `octavei` for his outstanding work, providing technical guidance and early-stage development in the Dota project.
+- `MockBL` Full-stack engineer with 7 years of experience in the blockchain industry, specializing in social, gaming, and DeFi products.
+- `lanyou` Software Test Engineer
+### Team Code Repos
+- https://github.com/DOTA-DOT20/dota-indexer
+- https://github.com/DOTA-DOT20/dota-indexer-db
+- https://github.com/DOTA-DOT20/dota-indexer-crawler
+
+- https://github.com/octavei/
+- https://github.com/jyaymusk/
+- https://github.com/MockBL/
+- https://github.com/DOTADurial
```

Comment:
> You mentioned in one of your comments that the team has "five years of experience in the Polkadot ecosystem". However, I don't see much activity on the Github profiles you linked. Could you elaborate?
Thank you very much to substrate for bringing us such a wonderful development experience. Personally, I have joined the development of substrate a long time ago and have my own project. It was 2019. The founder of the acala team introduced Polkadot technology to China and provided guidance and education on community technology. I was lucky to be the first beneficiary. Obviously, I changed my github account because different teams have different requirements for privacy and need us to keep our past experiences confidential.

Reply:
> > Thank you very much to substrate for bringing us such a wonderful development experience. Personally, I have joined the development of substrate a long time ago and have my own project. It was 2019. The founder of the acala team introduced Polkadot technology to China and provided guidance and education on community technology. I was lucky to be the first beneficiary. Obviously, I changed my github account because different teams have different requirements for privacy and need us to keep our past experiences confidential.